### ISSUE TITLE
Best hyperparameters for coco, pascal, nuswide and cub?

### ISSUE THREAD

**Issue by jameslahm** [snucml/LargeLossMatters#4]
Thank you for your great work!
Woud you mind sharing the best hyperparameters for coco, pascal, nuswide and cub under "End-to-End" training setting and "LinearInit" training setting? Thanks very much!

**Comment by jameslahm** [comment]
cc @kim-youngwook . Thanks very much!

**Comment by youngwk** [comment]
I will upload it after making arrangements for the experiment results.

**Comment by vanche** [comment]
Hi, I am interested in this task and your work. 
I would like to know if you could share the best hyperparameters for all datasets. Thank you for your awesome work. 
@kim-youngwook

**Comment by youngwk** [comment]
Sorry for being late.
Here is the best hyperparameter information.

1)  PASCAL VOC
SGD optimizer, learning rate : 5e-3
delta_rel : LL-R 0.2, LL-Ct 0.2, LL-Cp 0.1

2)  MS COCO
ADAM optimizer, learning rate : 1e-5
delta_rel : LL-R 0.4, LL-Ct 0.2, LL-Cp 0.2

3)  NUSWIDE
ADAM optimizer, learning rate : 1e-5
delta_rel : LL-R 0.4, LL-Ct 0.2, LL-Cp 0.2

4)  CUB
ADAM optimizer, learning rate : 7.5e-5
delta_rel : LL-R 0.4, LL-Ct 0.4, LL-Cp 0.2
